lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -912,21 +912,22 @@ def test_bypass_course_info(self):
 
         self.assertFalse(self.course.bypass_home)
 
-        response = self.client.get(reverse('info', args=[course_id]))
+        response = self.client.get(reverse('openedx.course_experience.course_home', args=[course_id]))
         self.assertEqual(response.status_code, 200)
 
-        response = self.client.get(reverse('info', args=[course_id]), HTTP_REFERER=reverse('dashboard'))
+        course_home_url = reverse('openedx.course_experience.course_home', args=[course_id])
+        response = self.client.get(course_home_url, HTTP_REFERER=reverse('dashboard'))
         self.assertEqual(response.status_code, 200)
 
         self.course.bypass_home = True
         self.store.update_item(self.course, self.user.id)  # pylint: disable=no-member
         self.assertTrue(self.course.bypass_home)
 
-        response = self.client.get(reverse('info', args=[course_id]), HTTP_REFERER=reverse('dashboard'))
+        response = self.client.get(course_home_url, HTTP_REFERER=reverse('dashboard'))
 
         self.assertRedirects(response, reverse('courseware', args=[course_id]), fetch_redirect_response=False)
```

Comment:
@robrap @dianakhuang this test and another test fail because there appears to be some redirect logic for course_info I believe around a course.bypass_home flag, and it seems this logic is not replicated for course home.
Who would have the product context to know if this feature was intentionally left out/is meant to be left out, or if it was an oversight and needs to be added?

Reply:
Taking a quick look at this, it seems as though that flag has been deprecated, so we should be able to say that the new course home doesn't obey that flag and delete any tests that assume this logic.